packages/grid/data-grid/src/tests/pagination.DataGrid.test.tsx

```diff
@@ -384,12 +383,12 @@ describe('<DataGrid /> - Pagination', () => {
 
       const footerHeight = document.querySelector('.MuiDataGrid-footerContainer')!.clientHeight;
       const expectedViewportRowsLengthBefore = Math.floor(
-        (heightBefore - DEFAULT_GRID_OPTIONS.headerHeight - footerHeight) /
-          DEFAULT_GRID_OPTIONS.rowHeight,
+        (heightBefore - DATA_GRID_DEFAULT_PROPS.headerHeight - footerHeight) /
```

Comment:
We have been setting manually the header and row height for the test. This file is the only one where the default value is used. I would use the props to set the value.

https://github.com/mui-org/material-ui-x/blob/f79a1fd0c32ddc9a7d2e1cdf87e81d369ce6827e/packages/grid/x-grid/src/tests/rows.XGrid.test.tsx#L317-L322